Aptoide/aptoide-client-v8
# Feature/mob 690 navigation between views

**What does this PR do?**

This pr aims at implementing navigation from email link to app login 

**Database changed?**

 No

**Where should the reviewer start?**


**How should this be manually tested?**

Run the deeplink (ask me for it), check that it returns an error (forced, at the moment).

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-690](https://aptoide.atlassian.net/browse/MOB-690)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
